qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_base.py

```diff
@@ -129,7 +130,9 @@ def from_model_name(model: str, slot_nr: int, parent: 'KeysightB1500',
         Returns:
             A specific instance of :class:`.B1500Module`
         """
-        if model == "B1517A":
+        if model == "B1511B":
+            return B1511B(slot_nr=slot_nr, parent=parent, name=name)
```

Comment:
in order to test that this works, we can change the pyvisa-sim yaml file to specify that one of the slots is B1511B as opposed to B1517A